plugins/vault/crons/processWithdrawals.js

```diff
@@ -325,8 +326,22 @@ const processWithdrawals = () => {
 									fields: ['transaction_id']
 								}
 							)
-								.then(() => {
-									return { success: true };
+								.then((data) => {
+									return {
+										success: true,
+										send: true,
+										data: {
+											email: withdrawal.User.email,
+											amount: getAmount(data.amount, data.fee),
```

Comment:
The fee is being subracted here the next `.then` block is reading the amount from this object being sent

Reply:
@abeikverdi